### ISSUE TITLE
RTC not actually updated after reboot

### ISSUE THREAD

**Issue by Pobega** [goweiwen/Allium#80]
I'm having a similar issue to https://github.com/goweiwen/Allium/issues/72, I've tried unplugging the and replugging the battery with no luck. I've got my hands on a Miyoo Mini+ with RTC and I'm unable to adjust the time, sometimes. It seems that occasionally I'm able to, but not always. My current issue is that I have the time _mostly_ correct but I need to use an incorrect UTC offset to get it to stick.

Basically when I make changes and reboot I end up with the same time as before the changes.

**Comment by goweiwen** [comment]
This should be fixed here: https://github.com/goweiwen/Allium/commit/3f7540b15dbc1401bb97cd7f807f9cc3e46f7f63#diff-ec26c54ab0d099830ec6b3737e6ce1df071fd54b7fba84b5e9f7f4e577727c69R76
